
+++ //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/userspace/private/apps/httpd/html/menuBcm.js
--- //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/userspace/private/apps/httpd/html/menuBcm.js

@@ -415,6 +415,7 @@
 		   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_MENU_SSID), 'wlrouter/ssid.asp'));
 		   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_MENU_SECURITY), 'wlrouter/security.asp'));
 		   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_MENU_WPS), 'wlrouter/wps.asp'));
+		   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_MENU_DPP), 'wlrouter/dpp.asp'));
 		   if(wlPasspoint == '1') {
 			   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_PASSPOINT), 'wlrouter/passpoint.asp'));
 		   }
@@ -851,6 +852,7 @@
 		   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_MENU_SSID), 'wlrouter/ssid.asp'));
 		   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_MENU_SECURITY), 'wlrouter/security.asp'));
 		   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_MENU_WPS), 'wlrouter/wps.asp'));
+		   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_MENU_DPP), 'wlrouter/dpp.asp'));
 		   if(wlPasspoint == '1') {
 			   insDoc(wl_wr_node, gLnk('R', getMenuTitle(MENU_WL_PASSPOINT), 'wlrouter/passpoint.asp'));
 		   }

+++ //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/userspace/private/apps/httpd/html/menuTitle.js
--- //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/userspace/private/apps/httpd/html/menuTitle.js

@@ -188,6 +188,7 @@
 MENU_ETH_LAG                 = 187
 MENU_TL_WAN_SELECT           = 188
 MENU_WL_IQOS                 = 189
+MENU_WL_MENU_DPP             = 190
 
 var menuTitle = new Array();
 menuTitle[MENU_DEVICE_INFO]                           = 'Device Info';
@@ -379,6 +380,7 @@
 menuTitle[MENU_ETH_LAG]                               = 'ETH LAG';
 menuTitle[MENU_TL_WAN_SELECT]                         = 'WAN Select';
 menuTitle[MENU_WL_IQOS]                               = 'iQoS';
+menuTitle[MENU_WL_MENU_DPP]                           = 'DPP';
 
 function getMenuTitle(index) {
    return menuTitle[index];
